net/portmapper/pcp_test.go

```diff
@@ -12,6 +12,8 @@ import (
 
 var examplePCPMapResponse = []byte{2, 129, 0, 0, 0, 0, 28, 32, 0, 2, 155, 237, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 129, 112, 9, 24, 241, 208, 251, 45, 157, 76, 10, 188, 17, 0, 0, 0, 4, 210, 4, 210, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 255, 255, 135, 180, 175, 246}
 
+var examplePCPDiscoResponse = []byte{2, 128, 0, 0, 0, 0, 0, 0, 0, 5, 39, 196, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
```

Comment:
I'd rather not hard-code this whole response packet. I'd rather see us build replies.